cvs diff: Diffing setup_native
cvs diff: Diffing setup_native/inc
cvs diff: Diffing setup_native/inc/setup_native
cvs diff: Diffing setup_native/prj
cvs diff: Diffing setup_native/registry
cvs diff: Diffing setup_native/registry/data
cvs diff: Diffing setup_native/registry/data/org
cvs diff: Diffing setup_native/registry/data/org/openoffice
cvs diff: Diffing setup_native/registry/data/org/openoffice/Office
cvs diff: Diffing setup_native/scripts
cvs diff: Diffing setup_native/scripts/source
cvs diff: Diffing setup_native/source
cvs diff: Diffing setup_native/source/java
cvs diff: Diffing setup_native/source/linux
cvs diff: Diffing setup_native/source/packinfo
Index: setup_native/source/packinfo/packinfo_office.txt
===================================================================
RCS file: /cvs/installation/setup_native/source/packinfo/packinfo_office.txt,v
retrieving revision 1.13
retrieving revision 1.13.8.1
diff -u -u -p -b -w -B -r1.13 -r1.13.8.1
--- setup_native/source/packinfo/packinfo_office.txt	21 Apr 2008 09:04:45 -0000	1.13
+++ setup_native/source/packinfo/packinfo_office.txt	6 May 2008 15:54:29 -0000	1.13.8.1
@@ -160,6 +160,21 @@ packageversion = "%OOOPACKAGEVERSION"
 End
 
 Start
+module = "gid_Module_Optional_Binfilter"
+solarispackagename = "ooobasis%WITHOUTDOTOOOBASEVERSION-binfilter"
+solarisrequires = "ooobasis%WITHOUTDOTOOOBASEVERSION-core01"
+packagename = "ooobasis%OOOBASEVERSION-binfilter"
+freebsdrequires = "ooobasis%OOOBASEVERSION-calc,ooobasis%OOOBASEVERSION-draw,ooobasis%OOOBASEVERSION-impress,ooobasis%OOOBASEVERSION-math,ooobasis%OOOBASEVERSION-writer"
+requires = "ooobasis%OOOBASEVERSION-calc,ooobasis%OOOBASEVERSION-draw,ooobasis%OOOBASEVERSION-impress,ooobasis%OOOBASEVERSION-math,ooobasis%OOOBASEVERSION-writer"
+copyright = "1999-2007 by OpenOffice.org"
+solariscopyright = "solariscopyrightfile"
+vendor = "OpenOffice.org"
+description = "Legacy filters (e.g. StarOffice 5.2) for OpenOffice.org %OOOBASEVERSION"
+destpath = "/opt"
+packageversion = "%OOOPACKAGEVERSION"
+End
+
+Start
 module = "gid_Module_Optional_Grfflt"
 solarispackagename = "ooobasis%WITHOUTDOTOOOBASEVERSION-graphicfilter"
 solarisrequires = "ooobasis%WITHOUTDOTOOOBASEVERSION-core01"
Index: setup_native/source/packinfo/packinfo_office_lang.txt
===================================================================
RCS file: /cvs/installation/setup_native/source/packinfo/packinfo_office_lang.txt,v
retrieving revision 1.6
retrieving revision 1.6.8.1
diff -u -u -p -b -w -B -r1.6 -r1.6.8.1
--- setup_native/source/packinfo/packinfo_office_lang.txt	18 Apr 2008 10:36:23 -0000	1.6
+++ setup_native/source/packinfo/packinfo_office_lang.txt	6 May 2008 15:54:29 -0000	1.6.8.1
@@ -167,6 +167,21 @@ packageversion = "%OOOPACKAGEVERSION"
 End
 
 Start
+module = "gid_Module_Langpack_Binfilter"
+solarispackagename = "ooobasis%WITHOUTDOTOOOBASEVERSION-%LANGUAGESTRING-binfilter"
+solarisrequires = "ooobasis%WITHOUTDOTOOOBASEVERSION-%LANGUAGESTRING"
+packagename = "ooobasis%OOOBASEVERSION-%LANGUAGESTRING-binfilter"
+provides = "ooobasis%WITHOUTDOTOOOBASEVERSION-%LANGUAGESTRING-binfilter"
+requires = "ooobasis%OOOBASEVERSION-%LANGUAGESTRING"
+copyright = "1999-2007 by OpenOffice.org"
+solariscopyright = "solariscopyrightfile"
+vendor = "OpenOffice.org"
+description = "Legacy filters (e.g. StarOffice 5.2) for OpenOffice.org %OOOBASEVERSION, language %LANGUAGESTRING"
+destpath = "/opt"
+packageversion = "%OOOPACKAGEVERSION"
+End
+
+Start
 module = "gid_Module_Langpack_Onlineupdate"
 solarispackagename = "ooobasis%WITHOUTDOTOOOBASEVERSION-%LANGUAGESTRING-onlineupd"
 solarisrequires = "ooobasis%WITHOUTDOTOOOBASEVERSION-%LANGUAGESTRING"
cvs diff: Diffing setup_native/source/ulfconv
cvs diff: Diffing setup_native/source/win32
cvs diff: Diffing setup_native/source/win32/customactions
cvs diff: Diffing setup_native/source/win32/customactions/indexingfilter
cvs diff: Diffing setup_native/source/win32/customactions/javafilter
cvs diff: Diffing setup_native/source/win32/customactions/languagepacks
cvs diff: Diffing setup_native/source/win32/customactions/patch
cvs diff: Diffing setup_native/source/win32/customactions/quickstarter
cvs diff: Diffing setup_native/source/win32/customactions/reg4msdoc
cvs diff: Diffing setup_native/source/win32/customactions/regactivex
cvs diff: Diffing setup_native/source/win32/customactions/regpatchactivex
cvs diff: Diffing setup_native/source/win32/customactions/regpython
cvs diff: Diffing setup_native/source/win32/customactions/relnotes
cvs diff: Diffing setup_native/source/win32/customactions/shellextensions
cvs diff: Diffing setup_native/source/win32/customactions/tools
cvs diff: Diffing setup_native/source/win32/nsis
cvs diff: Diffing setup_native/source/win32/stwrapper
